test/test_ops.py

```diff
@@ -477,6 +477,8 @@ def out_fn(output):
 
     @_alias_ops((op for op in op_db if op.aliases))
     def test_jit_alias_remapping(self, device, dtype, op):
+        tensor = torch.tensor
```

Comment:
Maybe add a comment here

Reply:
Ah, right. Thanks for the suggestion. I'm thinking of something like: `# Required to avoid undefined value: tensor error in JIT compilation of the function template.`

Does this sound good to you? @kshitij12345

Reply:
Update: have added the comment.